apps/xprof_gui/new-priv/src/actions/MonitoringActions.js

```diff
@@ -0,0 +1,29 @@
+import { remove } from 'lodash';
+import * as types from '../constants/ActionTypes';
+import { getMfas } from '../selectors/CommonSelectors';
+import { STOP_MONITORING_FUNCTION_URL } from '../constants/';
+import { callApi } from '../utils/ApiUtils';
+
+export const stopMonitoringFunctionRequest = mfas => ({
+  type: types.STOP_MONITORING_FUNCTION,
+  mfas,
+});
+
+export const stopMonitoringFunctionError = mfas => ({
+  type: types.STOP_MONITORING_FUNCTION_ERROR,
+  mfas,
+});
+
+export const stopMonitoringFunction = mfa => async (dispatch, getState) => {
+  const state = getState();
+  const mfas = getMfas(state);
+  const mfasReduced = remove(mfas, mfa);
+
+  dispatch(stopMonitoringFunctionRequest(mfasReduced));
+
+  const { error } = await callApi(`${STOP_MONITORING_FUNCTION_URL}?` +
```

Comment:
I have couple of issue with it :) 
I don't like this kind of string concatenation for url, especially that there is no urlencoding done. It might be a problem in the future when API evolves, so It would be nice to have an uniform way. 

TBH I'd create a module with API calls as functions instead one generic function and bunch of constants for endpoints. For example, from the end user perspective: 
`XprofApi.stopMonitoringFunction(module, function, arity);`

All the url construction and parameters joining can be done in this module it would generate less noise in function such as this one.  Additionally, you don't need to have URLs in globals module as everything will be used in one file, so the whole API won't leak through globals and I will be easy to change/extend the API.

As a by product we will have and SDK in case somebody wants to write alternative UI ;)